esptfa_arima/Test_Management/services/analysis_doc_service.py

```
@@ -91,6 +96,15 @@ def create_analysis_document(draft: TestDraft):
 
 
 
+# STARTING ARIMA MODEL
+def start_arima_model(document):
+    try:
+        arima_driver(document)
+        # process_analysis_document(document.analysis_document_id)
+    except Exception as e:
+        logger.error(f"Error starting ARIMA model: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)